app-frontend/src/app/components/colorCorrectScenes/colorCorrectScenes.html

```diff
@@ -44,6 +44,8 @@
           selected="$ctrl.isSelected(scene)"
           on-select="$ctrl.setSelected(scene, selected)"
           class="selectable"
+          ng-mouseover="$ctrl.$parent.setHoveredScene(scene)"
```

Comment:
The preferred way to for children to pass information back up to a parent is via an ampersand (`&`) output binding. The interaction between `colorCorrectAdjust` and its parent `colorCorrectPane` is a good example. There's also some more info available at https://docs.angularjs.org/guide/component .